doc/src/build/comms.md

```diff
@@ -2,12 +2,14 @@
 title: Connect and Communicate with the Sui Network
 ---
 
-Now that you have [installed Sui](install.md), [started the Sui network](cli-client.md), and learned how to [create smark contracts in Move](move/index.md) and [program Sui objects](programming-with-objects/index.md), it's time to let your apps talk to Sui. The pages in this section provide various options for communicating with Sui.
+Now that you have [installed Sui](install.md), [connected to Devnet](../explore/devnet.md), and started to [create smart contracts in Move](move/index.md) and [program Sui objects](programming-with-objects/index.md), it's time to let your apps talk to Sui.
 
-* Use the [Sui CLI client](cli-client.md) to start and set up the Sui network.
-* Set up your own [local Sui RPC server and use the Sui JSON-RPC API](json-rpc.md) to interact with a local Sui network.
+The pages in this section provide various options for communicating with Sui:
```

Comment:
title is talk, this says communicate, next says interact - can we align on one? Talk is potentially confusing for i18n

Reply:
> title is talk, this says communicate, next says interact - can we align on one? Talk is potentially confusing for i18n

Nav will always be shorter than full titles. Further, we should use synonyms to improve SEO and avoid repetition:
https://googleblog.blogspot.com/2010/01/helping-computers-understand-language.html

Remember that co-worker of yours who struggled because the term they sought wasn't included? Synonyms would have helped there, no?

Reply:
didn't look at the nav. I think this is not what they mean when suggesting using synonyms to drive SEO:
https://www.searchenginejournal.com/synonyms-seo/346026/#close

Reply:
> didn't look at the nav. I think this is not what they mean when suggesting using synonyms to drive SEO: https://www.searchenginejournal.com/synonyms-seo/346026/#close

Please explain